compatibility_lib/compatibility_lib/configs.py

```diff
@@ -75,23 +87,36 @@ def _format_url(repo_name, setuppy_path=''):
 WHITELIST_PKGS = PKG_LIST
 
 WHITELIST_URLS = {
-    _format_url('googleapis/google-cloud-python', 'asset'): 'gcloud',
-    _format_url('googleapis/google-cloud-python', 'automl'): 'gcloud',
-    _format_url('googleapis/google-cloud-python', 'dataproc'): 'gcloud',
-    _format_url('googleapis/google-cloud-python', 'dlp'): 'gcloud',
-    _format_url('googleapis/google-cloud-python', 'iot'): 'gcloud',
-    _format_url('googleapis/google-cloud-python', 'kms'): 'gcloud',
-    _format_url('googleapis/google-cloud-python', 'legacy/google-cloud'):
-        'gcloud',
-    _format_url('googleapis/google-cloud-python', 'ndb'): 'gcloud',
-    _format_url('googleapis/google-cloud-python', 'oslogin'): 'gcloud',
-    _format_url('googleapis/google-cloud-python', 'redis'): 'gcloud',
-    _format_url('googleapis/google-cloud-python', 'securitycenter'): 'gcloud',
-    _format_url('googleapis/google-cloud-python', 'tasks'): 'gcloud',
-    _format_url('googleapis/google-cloud-python', 'test_utils'): 'gcloud',
-    _format_url('googleapis/google-cloud-python', 'texttospeech'): 'gcloud',
```

Comment:
Why are these being deleted?

Reply:
These are updated not deleted. Actually wanted to confirm with you is the value in the whitelist dict representing the PyPI names of the packages? And why all of them are `'gcloud'`? I changed them to the corresponding `google-cloud-*` names.

Reply:
Oh sorry my bad - at a quick glance, the way the changes being displayed on github made me think they're being deleted.
Yes the dict values are the PyPI names. I think because the original list only included `gcloud` and not the google-cloud* names, I just assigned the dict values to be gcloud as well.

Reply:
’gcloud‘ and 'google-cloud-*' are different things actually :) The original list just didn't include all the google cloud client libraries, and I added them in this PR.